app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardViewModel.java

```diff
@@ -0,0 +1,107 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.ListUtils;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.services.ApiClientType;
+import com.kickstarter.services.apiresponses.ProjectsEnvelope;
+import com.kickstarter.ui.activities.CreatorDashboardActivity;
+
+import java.util.List;
+
+import rx.Notification;
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+public interface CreatorDashboardViewModel {
+  interface Inputs {
+    void projectViewClicked();
+  }
+
+  interface Outputs {
+
+    Observable<Project> latestProject();
+    Observable<String> projectBackersCountText();
+    Observable<String> projectNameTextViewText();
+    Observable<Pair<Project, RefTag>> startProjectActivity();
+    Observable<String> timeRemaining();
+  }
+
+  final class ViewModel extends ActivityViewModel<CreatorDashboardActivity> implements Inputs, Outputs {
+    private final ApiClientType client;
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+      this.client = environment.apiClient();
+
+      final Observable<Notification<ProjectsEnvelope>> projectsNotification =
+        this.client.fetchProjects(true).materialize().share();
+
+      final Observable<ProjectsEnvelope> projectsEnvelope = projectsNotification
+        .compose(Transformers.values());
```

Comment:
you can use static imports for `values()` here, same for all `Transformers.` functions